Libplanet/Action/ActionEvaluator.cs

```diff
@@ -233,14 +202,23 @@ internal static ValidatorSet NullValidatorSetGetter()
         /// <param name="tx">The <see cref="Transaction{T}"/> to evaluate.</param>
         /// <returns>An <see cref="IImmutableSet{T}"/> of updated <see cref="Address"/>es.
         /// </returns>
+        /// <typeparam name="T">An <see cref="IAction"/> type.  It should match
+        /// the <see cref="Block{T}"/>'s type parameter.</typeparam>
```

Comment:
Well.. I'm not sure but it is better to use `Transaction{T}` instead of `Block{T}` in this method.

```suggestion
        /// <typeparam name="T">An <see cref="IAction"/> type.  It should match
        /// the <see cref="Transaction{T}"/>'s type parameter.</typeparam>
```

Reply:
Fixed on https://github.com/planetarium/libplanet/pull/2703/commits/8ac312e3a74ae2b3fda68968bc7efb8c492dfc26.